pages/documentation/grid/model/index.md

```diff
@@ -10,150 +10,122 @@ latex: true
+
+The PowSyBl grid model allows a full representation of the substation connectivity where all the switching devices and busbars are defined (_node/breaker_ level). Automated topology calculation permits to obtain views of the network up to the _bus/branch_ level.
+
+Different states of the network can be stored together with the power system model in an efficient way. The set of attributes that define a given state of the network (steady state hypothesis and state variables) are collectively organized in _variants_. The user can create and remove _variants_ as needed. Setting and getting variant dependent attributes on network objects use the current _variant_.
+
+A set of PowSyBl networks can be merged together in a single network view, and sub-parts of the network model can be easily extracted as separate networks.
+
+All elements modeled in the network are identified through a unique ID, and optionally described by a name that is easier to interpret for a human. All components can be _extended_ by the user to incorporate additional structured data.
 
 ## Network core model
 
-### Network 
+In the following sections the different network components are described in terms of its main attributes and electro-technical representation. The attributes shared by all the network components are described in the next table:
+
+| Attribute | Description |
+| --------- | ----------- |
+| $$Id$$ | Unique Id assigned to each network component |
+| $$Name$$ | Human readable alphanumeric identifier |
+| $$Fictitious$$ | To identify non-physical network components |
+| $$Aliases$$ | Additional unique identifiers associated with each network component |
+| $$Properties$$ | To add additional data items to network components |
+
+The `Id` is the only required attribute and by default `Fictitious` is set to `false`. `Aliases` offers the possibility of adding additional unique identifiers to each component. An `AliasType` can be specified to indicate what it corresponds to. `Properties` allows to associate additional arbitrary data items under the general schema of pairs `<key, value>`.
+
+### Network
 [![Javadoc](https://img.shields.io/badge/-javadoc-blue.svg)](https://javadoc.io/doc/com.powsybl/powsybl-core/latest/com/powsybl/iidm/network/Network.html)
 
-In IIDM, the network is constituted of [substations](#substation), which are themselves constituted of [voltage levels](#voltage-level).
-All the equipments are then connected to the voltage levels.
-The network comprises metadata in IIDM: 
-- a case date: the date and time of the target network that is being modelled
-- a forecast distance: the number of minutes between the network generation date and the case date
+In the PowSyBl grid model, the Network contains [substations](#substation), which themselves contain [voltage levels](#voltage-level).
```

Comment:
I would keep the old presentation that is less technical: don't forget this is a functional documentation and it should be readable by non developer people

Reply:
Here maybe the table is not required.